Automattic/jetpack
# Updating getting extensions to allow for excluding extensions from an option

Fixes p1HpG7-6RI-p2

#### Changes proposed in this Pull Request:
This updates the final list of extensions retrieved to include whitelisted extensions minus a list of extensions to be excluded from options if the option is set.


#### Testing instructions:
Testing options:

- Add an option for jetpack_extensions_exclusions
- Include extensions to be excluded in the option
- Verify that the extensions excluded in the option are not available
- Run tests for ```test_get_available_extensions``` in  ```tests/php/test_class.jetpack-gutenberg.php```

#### Proposed changelog entry for your changes:
- Adding option to allow exclusion of whitelisted extensions from final extension list.
